src/components/Home.js

```diff
@@ -1,7 +1,151 @@
-import React from "react";
+import React, { useEffect } from "react";
+import Typography from "@material-ui/core/Typography";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import { connect } from "react-redux";
+import { getHackathonContent } from "../store/actions/hackathonActions";
 
-const Home = () => {
-  return <h1>Home page</h1>;
-};
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "center",
+    color: theme.palette.text.secondary
+  }
+}));
+
+export const Home = ({
+  from,
+  to,
+  dispatch,
+  hackathon,
+  title,
+  information,
+  status,
+  theme,
+  price
+}) => {
+  const classes = useStyles();
+  const scheduleCard = (
```

Comment:
These cards all look quite similar to me, why not create a "clever" layout component?

```
const ValueListCard = ({ id, title, values }) => (
  <Card data-testid={`${id}-card` style={{ width: "100%" }}>
    <CardContent>
      <Typography color="textSecondary" gutterBottom>
        {title}
      </Typography>

      {values.map((value, index) => (
        <Typography key={index} variant="body2" component="p">
          {value}
        </Typography>
      )
    </CardContent>
  </Card>
)
```

then you can use it like so

```
<ValueListCard id="schedule" title="Schedule" values={[`From: ${from}`, `To: ${to}`]} />
<ValueListCard id="price" title="Price" values={[price]} />
```

Reply:
passing data in values in [ ] was really cool, lesson learned, thank you